org.eclipse.wildwebdeveloper.xml/plugin.xml

```diff
@@ -2,6 +2,7 @@
 <?eclipse version="3.4"?>
 <plugin>
    <extension-point id="xmllsExtension" name="%ExtPoint.xmllsExtension" schema="schema/xmllsExtension.exsd"/>
+   <extension-point id="xmllsClasspathExtensionProvider" name="%ExtPoint.xmllsClasspathExtensionProvider" schema="schema/xmllsClasspathExtensionProvider.exsd"/>
```

Comment:
I think it would rather be the same extension point, but allowing different children element (`jar` and `classpathProvider`). As both serve the same goal, it's more consistent to use the same extension point.

Reply:
Agreed, I'll try to fix this.